letshelp-letsencrypt/setup.py

```diff
@@ -7,7 +7,7 @@
 version = '0.5.0.dev0'
 
 install_requires = [
-    'setuptools',  # pkg_resources
+    'setuptools>=1.0',  # pkg_resources
```

Comment:
We don't need to specify a version here because `letshelp-letsencrypt` doesn't use `cryptography`.